crates/re_viewer/src/ui/memory_panel.rs

```diff
@@ -118,9 +120,14 @@ impl MemoryPanel {
             Self::tracking_stats(ui, tracking_stats);
         } else {
             ui.separator();
-            ui.label(format!(
-                "Set {RERUN_TRACK_ALLOCATIONS}=1 to turn on detailed allocation tracking."
-            ));
+            if ui.button("Turn on detailed allocation tracking").clicked() {
+                re_memory::accounting_allocator::set_tracking_callstacks(true);
```

Comment:
any reason this is not a checkbox? No directly feedback of the interaction and no turning off right now

Reply:
Oh yeah, I had that thought and forgot about it 😅